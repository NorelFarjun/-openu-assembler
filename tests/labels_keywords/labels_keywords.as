.extern stop
.entry r4
  
r7: .data 1
_Bad: rts

; This is fine here
r9: .data 0
