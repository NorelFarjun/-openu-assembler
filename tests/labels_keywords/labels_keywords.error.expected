0: Incorrect label name 'stop'
1: Incorrect label name 'r4'
3: bad label name 'r7'
4: Incorrect instruction line
Bad input file - not outputting
